### PR TITLE
Add support to retrieve all API keys if user has privilege.

### PR DESCRIPTION
**Pull request by codebrain**
Test to assert server returns a response when no parameters are specified.

Implements https://github.com/elastic/elasticsearch/pull/47274

### CODE REVIEW

File: src/Tests/Tests/XPack/Security/ApiKey/SecurityApiKeyTests.cs [hunk 1]
```diff
@@ -120,6 +121,25 @@ public SecurityApiKeyTests(XPackCluster cluster, EndpointUsage usage) : base(new
 						(v, c, r) => c.Security.CreateApiKeyAsync(r)
 					)
 			},
+			{
```
Comment (Mpdreamz) [inline src/Tests/Tests/XPack/Security/ApiKey/SecurityApiKeyTests.cs]: This needs to be denoted with a version somehow (maybe an additional parameter taking a version range string?)

**Comment by Mpdreamz** [comment]
Failing tests relate to #4280  and `ProcessClusterEventTimeoutException`

**Comment by codebrain** [comment]
Nice addition @Mpdreamz - merging now.

**Comment by codebrain** [comment]
Ported to `master`

**Pull request merged**